NitroxClient/GameLogic/Terrain.cs

```diff
@@ -93,5 +94,19 @@ private IEnumerator WaitAndSyncCells(float delay)
                 yield return new WaitForSeconds(0.05f);
             }
         }
+
+        public IEnumerator WaitForWorldLoad()
```

Comment:
WaitForWorldLoad can be static to remove the need of storing an instance and since it's only accessing static singleton